libmscore/chord.cpp

```diff
@@ -2523,11 +2530,11 @@ Element* Chord::drop(EditData& data)
 
             case ElementType::TREMOLO:
                   {
-                  Tremolo* t = toTremolo(e);
+                  Tremolo* t = static_cast<Tremolo*>(e);
```

Comment:
Why? `toXxxx()` is the new style to be used in master

Reply:
I think that part have merge problems. I did not change this code. At fact in chord.cpp i just add calculation of bends bounding boxes.

Reply:
This is still an issue.